plugins/assemblyai/src/stt.ts

```diff
@@ -0,0 +1,197 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+// SPDX-FileCopyrightText: 2024 Josiah Bryan, LLC
+//
+// SPDX-License-Identifier: Apache-2.0
+import { type AudioBuffer, AudioByteStream, AudioEnergyFilter, log, stt } from '@livekit/agents';
+import type { AudioFrame } from '@livekit/rtc-node';
+import { AssemblyAI } from 'assemblyai';
+import type { RealtimeTranscriber } from 'assemblyai';
+
+export interface STTOptions {
+  apiKey?: string;
+  interimResults: boolean;
+  sampleRate: number;
+  keywords: [string, number][];
+  endUtteranceSilenceThreshold?: number;
+}
+
+const defaultSTTOptions: STTOptions = {
+  apiKey: process.env.ASSEMBLY_AI_KEY,
+  interimResults: true,
+  sampleRate: 16000,
+  keywords: [],
+  // NOTE:
+  // The default is 700ms from AssemblyAI.
+  // I use a low default of 300ms here because I also use
+  // the new end-of-utterance model from LiveKit to handle
+  // turn detection in my agent. Which means that even though
+  // this will quickly return a final transcript EVEN THOUGH
+  // USER IS NOT DONE SPEAKING, the EOU model from LiveKit
+  // DOES properly differentiate and doesn't interrupt (magically!)
+  // Ref: https://blog.livekit.io/using-a-transformer-to-improve-end-of-turn-detection/
```

Comment:
we can drop this in favour of a simpler note, as the implementation detail is not relevant to this plugin:
```suggestion
  // AssemblyAI default is 700ms
```

also, you wrote 300ms but the threshold in code looks to be 200?

Reply:
Done, changed comment and value to 700